eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -218,6 +218,11 @@
           "new": "method void com.azure.spring.cloud.resourcemanager.provisioning.ServiceBusProvisioner::provisionTopic(java.lang.String, java.lang.String, com.azure.spring.cloud.resourcemanager.provisioning.properties.ServiceBusTopicProperties)",
           "justification": "New method added to support more properties."
         },
+        {
+          "code": "java.method.addedToInterface",
+          "new": "method void com.azure.spring.cloud.appconfiguration.config.AppConfigurationRefresh::refreshAsync()",
+          "justification": "Method was added to an interface."
+        },
```

Comment:
I do not see AppConfigurationRefresh changes in this PR. Also, this is a valid issue and should be fixed. We should not add new methods to existing interfaces. If required, we should add `default` methods.

Reply:
@srnagar @saragluna I've added default methods in a but fix PR I have up https://github.com/Azure/azure-sdk-for-java/pull/41680.

Reply:
@mrm9084 I have added a few comments on the PR https://github.com/Azure/azure-sdk-for-java/pull/41680.

Reply:
@srnagar , I added Matt's changes here to make this pr merge first.
FYI @mrm9084